documentation/dev-portal/src/nymvpn/gui.md

```diff
@@ -0,0 +1,40 @@
+# NymVPN Application (GUI)
+
+```admonish info
+Our alpha testing round is done with participants at live workshop events. This guide will not work for everyone, as the NymVPN source code is not yet publicly accessible. The alpha testing is done on Nym testnet Sandbox environment, this configuration is limited and will not work in the future.
+
+**If you commit to test NymVPN alpha, please start with the [user research form](https://opnform.com/forms/nymvpn-user-research-at-37c3-yccqko-2) where all the steps will be provided**. If you disagree with any of the conditions listed, please leave this page.
```

Comment:
ditto to question above re reusing the 37c3 form - if we are then ignore this

Reply:
yes, this is the latest form